frontend/src/scenes/funnels/FunnelLineGraph.tsx

```diff
@@ -30,20 +29,26 @@ export function FunnelLineGraph({
             inSharedMode={!!inSharedMode}
             showPersonsModal={showPersonsModal}
             tooltip={{
-                showHeader: false,
+                showHeader: true,
                 hideColorCol: true,
-                renderSeries: (_, datum) => {
-                    if (!steps?.[0]?.days) {
-                        return 'Trend'
-                    }
+                renderSeries: function renderSeries() {
                     return (
-                        getFormattedDate(steps[0].days?.[datum.dataIndex], filters.interval) +
-                        ' ' +
-                        shortTimeZone(insight.timezone)
+                        <div style={{ paddingTop: 20, height: 68 }}>
+                            Percentage
+                            <br />
+                            Entered/completed
+                        </div>
                     )
                 },
-                renderCount: (count) => {
-                    return `${count}%`
+                renderCount: function renderCount(count, datum) {
+                    if (!datum.reached_from_step_count || !datum.reached_to_step_count) {
+                        return null
+                    }
```

Comment:
This introduces a bug where series labels are displayed but counts are omitted where percentage is 0.

<img width="467" alt="Screen Shot 2022-06-09 at 1 37 21 PM" src="https://user-images.githubusercontent.com/13460330/172910184-10e37136-3dd2-4e0b-9e0f-cb652d37f2c1.png">